index.bs

```diff
@@ -1278,7 +1279,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
                             <code>|assertionCreationData|.[=assertionCreationData/signatureResult=]</code>.
 
                         :   {{AuthenticatorAssertionResponse/userHandle}}
-                        ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of
+                        ::  If <code>|assertionCreationData|.[=assertionCreationData/userHandleResult=]</code> is null, set this
+                            field to null. Otherwise, set this field to a new {{ArrayBuffer}}, created using |global|'s
```

Comment:
nit: I'm not sure these two sentences are technically correct given that the dfn of [ArrayBuffer](https://heycam.github.io/webidl/#idl-ArrayBuffer) is: "An _object_ that holds a pointer (which may be null) to a buffer of a fixed number of bytes". I.e., may need to only instantiate the arraybuffer and assign the bytes of userHandleResult to it, and if userHandleResult is null, then the buffer pointer is automagically set to null.

However, the present language may be close enough to pass muster with reviewers.

Reply:
As far as I could tell when I looked into it, it seemed like `ArrayBuffer` is a nullable type, if that's what you mean. Did I get that wrong?

Reply:
Or, rather, it looked to me making a field of type `ArrayBuffer` nullable is legal. In that case I suppose the value could be null in two ways, so you may be right in that it's more correct to keep it not nullable, but I feel like making the field itself explicitly nullable better communicates the intent. What do you think?